engine/src/cmd/armed.h

```diff
@@ -42,7 +42,7 @@ class Armed
     float gunspeed;
     float gunrange;
     float missilerange;
-    char  turretstatus;
+    char  turretstatus = 0;
```

Comment:
This should be done in the implementation file, not the header.

Reply:
Hmm. I'm not sure I agree. See https://stackoverflow.com/a/11594963/5067822

Reply:
@stephengtuggy I read that, but I'm not sure I agree. I don't necessarily see all new things in C++11/14/17 as advantageous or worthwhile.

Reply:
Well, we need to make sure that every variable and every field is initialized before it is used. (With a few exceptions where we actually want random values.) Where better to initialize these fields than at the point where they are declared?

If not here, then we should make sure to initialize every field in each constructor, in an initializer list. And that is likely to be repetitive and error-prone, seems to me.

Reply:
If we break things down properly it won't be so overwhelming for each constructor to initialize its variables.